#!/usr/bin/env python
PACKAGE = "project"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#				name, 			type, 	 level, 	description, 		default, 		min, 		max
gen.add("scaledDepthVarTH",    	float_t,   	 0, "scaledDepthVarTH parameter", 	-2.5,  	-10, 		10	)
gen.add("absDepthVarTH",    	float_t,   	 0, "absDepthVarTH parameter", 		 1.0,  	  0, 		5	)
gen.add("minNearSupport",    	  int_t,   	 0, "minNearSupport parameter", 	   7,  	  0, 		20	)
gen.add("sparsifyFactor",    	  int_t,   	 0, "sparsifyFactor parameter", 	   1,  	  0, 		5	)

#							Node Name , File Name		
exit(gen.generate(PACKAGE, "project", "project"))